src/Appwrite/Platform/Workers/StatsUsage.php

```diff
@@ -162,13 +300,20 @@ public function action(Message $message, callable $getProjectDB, callable $getLo
         $this->stats[$projectId]['project'] = $project;
         $this->stats[$projectId]['receivedAt'] = DateTime::now();
         foreach ($payload['metrics'] ?? [] as $metric) {
+            $metricKey = $metric['key'] ?? '';
+
+            // Skip invalid metrics
+            if (!$this->isValidMetric($metricKey)) {
+                throw new Exception('Invalid metric key: ' . $metricKey);
```

Comment:
shouldn't this log and skip, instead of throwing exception? don't throwing exception exit this foreach loop missing other valid metrics?